### ISSUE TITLE
verdi data cif list should use querybuilder in sqlalchemy

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida_core#901]
```
$ verdi data cif list
11/08/2017 06:26:47 PM, alembic.runtime.migration: [INFO] Context impl PostgresqlImpl.
11/08/2017 06:26:47 PM, alembic.runtime.migration: [INFO] Will assume transactional DDL.
Traceback (most recent call last):
  File "/Users/leopold/Applications/miniconda3/envs/aiida_master/bin/verdi", line 9, in <module>
    sys.exit(run())
  File "/Users/leopold/Applications/aiida_core/aiida/cmdline/verdilib.py", line 1049, in run
    aiida.cmdline.verdilib.exec_from_cmdline(sys.argv)
  File "/Users/leopold/Applications/aiida_core/aiida/cmdline/verdilib.py", line 1034, in exec_from_cmdline
    CommandClass.run(*argv[command_position + 1:])
  File "/Users/leopold/Applications/aiida_core/aiida/cmdline/baseclass.py", line 137, in run
    function_to_call(*args[1:])
  File "/Users/leopold/Applications/aiida_core/aiida/cmdline/baseclass.py", line 190, in run
    function_to_call(*args[1:])
  File "/Users/leopold/Applications/aiida_core/aiida/cmdline/commands/data.py", line 88, in list
    entry_list = self.query(parsed_args)
  File "/Users/leopold/Applications/aiida_core/aiida/cmdline/commands/data.py", line 1533, in query
    object_list = self.dataclass.query(q_object).distinct().order_by('ctime')
  File "/Users/leopold/Applications/aiida_core/aiida/orm/implementation/sqlalchemy/node.py", line 153, in query
    raise NotImplementedError("The node query method is not supported in "
NotImplementedError: The node query method is not supported in SQLAlchemy. Please use QueryBuilder.
```

**Comment by szoupanos** [comment]
This is important enough. 
I put as milestone 0.10.0 and if it is not ready in time we shift it to 0.10.1

**Comment by waychal** [comment]
Once we switch from pycifrw to pycodcif (as discussed in #882) check this again.

**Comment by szoupanos** [comment]
Fixed by PR #1007
